tools/upload_dataset.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Zips and uploads all files in <directory> to the dataset GCS bucket
+
+USAGE="Usage: $0 <dataset_directory>"
```

Comment:
Would this allow anyone who collected a dataset to upload to the rest (i.e. appending the observations to the ones in the GCS bucket)?

Reply:
since there's no sensitive information in our storage bucket, everyone has read access to it; but currently, only I have write access (we shouldn't open it up to everyone since it could be abused by people looking for free storage). It's super easy for me to add collaborators on the project as writers though (just need a gmail address)

Reply:
Okay, that makes sense.
Good to know!